### PR TITLE
WT-6068 Re-enable tests temporarily disabled during durable history development

### PR DESCRIPTION
**Pull request by lukech**
Re-enable Evergreen tasks that were previously disabled during DH development, and verified passing in a recent patch build. 

For those failed tasks in the same patch build, we have separate tickets created (under PM-1852) to address the effort of fixing and re-enabling them.

### CODE REVIEW

**Comment by tetsuo-cpp** [comment]
@lukech I think these tests might also be commented out in the `Makefile.am`. Can you please uncomment them in this PR too?

File: test/evergreen.yml [hunk 1]
```diff
@@ -2260,19 +2224,16 @@ buildvariants:
...
-    # Temporarily disabled
-    # - name: linux-directio
-    #   distros: ubuntu1804-build
+    - name: linux-directio
+      distros: ubuntu1804-build
     - name: syscall-linux
     - name: make-check-asan-test
     - name: configure-combinations
-    # Temporarily disabled
     # - name: checkpoint-filetypes-test
```
Comment (tetsuo-cpp) [inline test/evergreen.yml:2232]: How about these? I saw you uncommented the job above.

**Comment by lukech** [comment]
Thanks for the comment @tetsuo-cpp. I re-enabled tests in Makefile.am using 3c31d34.

File: test/evergreen.yml (hunk 1 above)
Comment (lukech) [reply test/evergreen.yml:2232]: The test is meant to be kept disabled. WT-6063 was created to address and the failure and re-enable it. I uncomment the task definition segment and left the task commented out at the build variant level, which in fact will achieve the same effect of disabling the task. The benefit is next time  when the test is ready to be enabled we only need 1-line change.

**Comment by lukech** [comment]
The `checkpoint-test` and `format-test` were a bit flaky - they failed in the latest PR build though succeeded in the previous ones. The failure in `checkpoint-test` is novel, and I'll keep it disabled and open a separate ticket to enable it later. The failure in `format-test` had been tracked by WT-5839. I'd prefer to re-enable it.

**Pull request merged**